scanner/tag_scanner.go

```diff
@@ -326,9 +327,30 @@ func (s *TagScanner) deleteOrphanSongs(ctx context.Context, dir string, tracksTo
 	return numPurgedTracks, nil
 }
 
+type artistItem struct {
+	artistId string
+}
+
+func (s *TagScanner) execute(workload interface{}) {
+	es := core.NewExternalMetadata(s.ds)
+	ctx := context.Background()
+	item := workload.(artistItem)
+	_, err := es.UpdateArtistInfo(ctx, item.artistId, 0, false)
+	if err != nil {
```

Comment:
I added a checkpoint here to investigate the code, but the debugger doesn't halt here.
`WARN[0002]: data not found`
![artist_view_error](https://user-images.githubusercontent.com/48797475/113753186-70653180-972b-11eb-988e-83dd7e08740f.png)

Reply:
I am investigating on this